<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="WINCTRL-BKLIGP2" failures="2" tests="2" name="Test(failed)" time="0.142" errors="0" timestamp="2023-01-14T15:43:33 IST">
  <testcase classname="TestNG.Demo4" name="TC6" time="0.026">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNG.Demo4.TC6(Demo4.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TC6 -->
  <testcase classname="TestNG.Demo3" name="TC3" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNG.Demo3.TC3(Demo3.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TC3 -->
  <testcase classname="TestNG.Demo4" name="TC5" time="0.0">
    <ignored/>
  </testcase> <!-- TC5 -->
  <testcase classname="TestNG.Demo3" name="TC4" time="0.0">
    <ignored/>
  </testcase> <!-- TC4 -->
</testsuite> <!-- Test(failed) -->
